src/app/model/Mission.ts

```diff
@@ -50,8 +50,8 @@ const defaultMissionData: MissionInterface = {
   recipientName: "No Recipient Name",
   recipientPhoneNumber: "",
   recipientId: "No Recipient Id", // reference?
-  created: new Date(), // time stamp
-  lastUpdated: new Date(), // time stamp
+  created: "", // time stamp
+  lastUpdated: "", // time stamp
```

Comment:
Haven't had a chance to fully review the models, will changing the model defaults to an empty string affect other components asking for this?

Reply:
it just not work with Date...that is why we opted for string